ci-operator/config/openshift/router/openshift-router-master.yaml

```diff
@@ -140,6 +140,26 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-upgrade-azure
+- always_run: false
+  as: perfscale-aws-ingress-perf
+  optional: true
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "9"
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      LOKI_USE_SERVICEMONITOR: "false"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
+      SET_ENV_BY_PLATFORM: custom
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    test:
+    - chain: openshift-qe-ingress-perf
+  timeout: 5h0m0s
```

Comment:
@jtaleric do you know how long the test usually runs?  It might be better to interrupt it sooner than 5 hours if it normally only takes 3, for example.

Reply:
The workload itself takes 1-1.5hr -- deployment is another 30-50min... and all the collection prow does (must-gathers). 5hr is a good buffer in case something takes a bit longer.